bolt-gcc-stage1.bash

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export TOPLEV=~/toolchain/gcc
+mkdir ${TOPLEV}
+cd ${TOPLEV}
+
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata/cc1
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata/cc1plus
+cd ${TOPLEV}/bolt-gcc
+GCCPATH=/usr/lib/gcc/x86_64-pc-linux-gnu/12.1.1
```

Comment:
do at least all linuxes install gcc there? i think there is some divergence on distros.

Would a `GCCPATH=$(which gcc)` work?

Reply:
and `readlink` to resolve the simlink instead?
It would be nice to not have it hardcoded here to a specific GCC version